app/components/pages/SubmissionWizard/index.js

```diff
@@ -13,104 +14,111 @@ import { schema as editorsPageSchema } from './steps/Editors/schema'
 import { schema as disclosurePageSchema } from './steps/Disclosure/schema'
 import WizardStep from './WizardStep'
 import ErrorPage from '../../../components/pages/Error'
+import { ON_UPLOAD_PROGRESS } from './operations'
 
 const SubmissionWizard = ({ match, history }) => (
   <SubmissionOperations manuscriptId={match.params.id}>
     {({ initialValues, updateManuscript, submitManuscript }) => (
-      <Switch>
-        <Route
-          exact
-          path={`${match.path}/files`}
-          render={() => (
-            <WizardStep
-              component={FilesPageContainer}
-              handleAutoSave={updateManuscript}
-              handleButtonClick={updateManuscript}
-              history={history}
-              initialValues={initialValues}
-              nextUrl={`${match.url}/submission`}
-              previousUrl={`${match.url}/author`}
-              step={1}
-              title="Write your cover letter and upload your manuscript"
-              validationSchema={filesPageSchema}
+      <Subscription subscription={ON_UPLOAD_PROGRESS}>
```

Comment:
The change in the PR - moving the subscription up into the Submission Wizard level instead of the File Upload component itself - means that this product requirement is now satisfied:
> During upload if the user navigates back a step in the wizard then returns to upload step, latest progress percentage displays or the success / error state should display

So it would be good for this PR to also include a test that covers this scenario?

Reply:
I've left testing for this out of this PR as I think it would fit into an issue for adding more Submission Wizard tests as well as extending the browser test with regards to uploading.

Reply:
Sure thing. I don't blame you for wanting to get the refactor in, so that we can unblock the other issues, if nothing else.

But do you not think it's good to write tests as we go?